### ISSUE TITLE
hello i have many lua files like this and  need a decompiler i will pay for this work

### ISSUE THREAD

**Issue by matrixhax** [viruscamp/luadec#68]
hello  i have this kind of lua files 
[https://cdn.discordapp.com/attachments/465472550193856526/736522199954882620/LuaPanda.lua](url)
i try all possible decompilers but not success to decompile it 
i need to have a decompiler for this lua files cause i have many of them
i wold rquest to make that decompiler for me and iw ill pay for it 
i hope you can reply me here 
or add my discord : MATRIX HAX#4074

**Comment by tst2005** [comment]
Is it legal ? How much do you pay ?

**Comment by tst2005** [comment]
@matrixhax I asked the FBI and got the decompiled sources with comment !
make me a offer 0$ should be good :D

**Comment by matrixhax** [comment]
> @matrixhax I asked the FBI and got the decompiled sources with comment !
> make me a offer 0$ should be good :D

hello bro sorry i didnt see the githup long time 
and yes i still want to decompile this files
i need a decompiler and i will pay any price

**Comment by tst2005** [comment]
Challenge accepted :-D

**Comment by tst2005** [comment]
@matrixhax we can start [here](https://zerobin.net/?c6d716422eb49c87#OcGgRudxksmoOQKpB5Ik3p6pWL7Rq5hew/3pOceU0fY=).

**Comment by matrixhax** [comment]
> @matrixhax we can start here.

bro that link expaired i cant continue talking

**Comment by matrixhax** [comment]
> > @matrixhax we can start here.
> 
> bro that link expaired i cant continue talking 
imean it expaired again
or you deleted the chat ?

**Comment by tst2005** [comment]
@matrixhax Yes I delete the initial link. I hope you remind the password and please bookmark this [link](https://zerobin.net/?efc16a051c940d50#/YwUu3CZqjwwwY1y4N7omGwvuJtVDmX2UuA5ii+LVks=) i will remove my answer soon.

**Comment by matrixhax** [comment]
> @matrixhax Yes I delete the initial link. I hope you remind the password and please bookmark this [link](https://zerobin.net/?efc16a051c940d50#/YwUu3CZqjwwwY1y4N7omGwvuJtVDmX2UuA5ii+LVks=) i will remove my answer soon.

bro i replaid you yesterday at morning and still waiting your reply  thank you

**Comment by matrixhax** [comment]
> Challenge accepted :-D

bro  i replaid you at the link again please read it 
i dont want to bother you but let us talk at least

**Comment by matrixhax** [comment]
please reply me bro 
i also delted old comments to let you feel better and i will delete new comments too
you can also delte yours too

**Comment by matrixhax** [comment]
bro please dont keep me waiting 
you dont lose anything and i am offering more than500 $
it is not illegal and it will not be shared it only for me bro
it is just a simple decompiler request

**Comment by matrixhax** [comment]
> > Challenge accepted :-D
brother at least give me the current decompiler you used to decompile the files you sent me
please

**Comment by tst2005** [comment]
About this file, I think https://github.com/Tencent/LuaPanda/blob/master/Debugger/LuaPanda.lua should be what you want.

**Comment by matrixhax** [comment]
> About this file, I think https://github.com/Tencent/LuaPanda/blob/master/Debugger/LuaPanda.lua should be what you want.

Bro i am not a developer i dont know what is this 
if it possible give me the decompiler you used to decompile the files you sent to me
and bro if you really can complete my task i will follow your rules as you want
and you can ask for higher price if you want and if it about trust i can send half money before you do the task
then you can trust me 
i really didnt find any one else can help me do this task so please help me out bro
